slick.grid.js

```diff
@@ -161,32 +157,36 @@ if (typeof Slick === "undefined") {
     var vScrollDir = 1;
 
     // private
+    const u = Slick.Utils;
```

Comment:
same as earlier comments, I would prefer to see `utils` as the variable name, if it doesn't conflict with anything else that is

Reply:
Done